### ISSUE TITLE
Semantical error in diagram in section 6.3

### ISSUE THREAD

**Issue by anjapetkovic** [pearcej/cpp4python#231]
In section about file handling, specifically in section 6.3 (https://runestone.academy/ns/books/published/cpp4python/Input_and_Output/InputandOutput.html)

in the second picture : `out_stream` is pointing to `myFile` instead of `anotherFile`. The text by the picture is correct though.

**Comment by Austing767** [comment]
@pearcej, Do you have a picture to fix this?

**Comment by pearcej** [comment]
@Austing767 No, you would need to edit the image itself or create a new one.  Be careful with the fonts though.
